packages/core/app/styles/navi-core/common/spinner.less

```diff
@@ -12,7 +12,7 @@
   width: 100%;
 }
 
-.loading-message {
+.navi-loading-message {
```

Comment:
Can this be in `navi-loading-message.less`?

Reply:
done!